packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -28,14 +29,15 @@ export const createEditorView = (
   accountId: string,
   pageEntityId: string,
   blocks: BlocksMap,
+  documentRootRef: RefObject<HTMLDivElement>,
```

Comment:
This contains the same node passed in the `renderNode` argument. We should de-duplicate.

Reply:
My bad! Didn't realise we were already passing the element there. Done in 248f0e73932a1b7b040fabf10865e4a6521914d1